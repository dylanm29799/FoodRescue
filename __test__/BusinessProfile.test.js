// __tests__/BusinessProfile.test.js

import React from "react";
import BusinessProfile from "../BusinessScreens/BusinessProfile";
import renderer from "react-test-renderer";
import { firebaseConfig } from "../config";
import * as firebase from "firebase";

firebase.initializeApp(firebaseConfig);
test("renders correctly", () => {
  const tree = renderer.create(<BusinessProfile />).toJSON();
  expect(tree).toMatchSnapshot();
});
