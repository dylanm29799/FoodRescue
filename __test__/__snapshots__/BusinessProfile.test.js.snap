// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "flex": 1,
      "justifyContent": "center",
      "width": "70%",
    }
  }
>
  <View
    style={
      Object {
        "justifyContent": "center",
        "paddingBottom": 64.28571428571428,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "MonM",
          "fontSize": 25.714285714285715,
          "padding": 6.428571428571429,
          "paddingTop": 21.428571428571427,
        }
      }
    >
       Business Name
    </Text>
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderColor": "#c44536",
          "borderRadius": 5,
          "borderWidth": 2,
          "fontFamily": "MonM",
          "fontSize": 32.14285714285714,
          "justifyContent": "center",
          "padding": 6.428571428571429,
          "paddingLeft": 15,
        }
      }
      underlineColorAndroid="transparent"
    >
      
    </TextInput>
    <Text
      style={
        Object {
          "fontFamily": "MonM",
          "fontSize": 25.714285714285715,
          "padding": 6.428571428571429,
          "paddingTop": 21.428571428571427,
        }
      }
    >
       Your Phone Number:
    </Text>
    <TextInput
      allowFontScaling={true}
      keyboardType="numeric"
      onChangeText={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderColor": "#c44536",
          "borderRadius": 5,
          "borderWidth": 2,
          "fontFamily": "MonM",
          "fontSize": 32.14285714285714,
          "justifyContent": "center",
          "padding": 6.428571428571429,
          "paddingLeft": 15,
        }
      }
      underlineColorAndroid="transparent"
    >
      
    </TextInput>
    <Text
      style={
        Object {
          "fontFamily": "MonM",
          "fontSize": 25.714285714285715,
          "padding": 6.428571428571429,
          "paddingTop": 21.428571428571427,
        }
      }
    >
       Email:
    </Text>
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderColor": "#c44536",
          "borderRadius": 5,
          "borderWidth": 2,
          "fontFamily": "MonM",
          "fontSize": 32.14285714285714,
          "justifyContent": "center",
          "padding": 6.428571428571429,
          "paddingLeft": 15,
        }
      }
      underlineColorAndroid="transparent"
    >
      
    </TextInput>
    <Text
      style={
        Object {
          "fontFamily": "MonM",
          "fontSize": 25.714285714285715,
          "padding": 6.428571428571429,
          "paddingTop": 21.428571428571427,
        }
      }
    >
      What's your category?
    </Text>
    <View
      style={
        Array [
          Object {
            "height": 40,
            "width": "100%",
          },
          Object {
            "zIndex": 5000,
          },
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onLayout={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#fafafa",
            "borderBottomLeftRadius": 5,
            "borderBottomRightRadius": 5,
            "borderColor": "#dfdfdf",
            "borderTopLeftRadius": 5,
            "borderTopRightRadius": 5,
            "borderWidth": 1,
            "flex": 1,
            "flexDirection": "row",
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 5,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "red",
              },
              Object {},
              Object {
                "flex": 1,
                "opacity": 1,
              },
              Object {
                "marginLeft": 5,
                "marginRight": 0,
              },
              false,
              [Function],
              Object {},
            ]
          }
        >
          Select an item
        </Text>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomLeftRadius": 0,
                "borderTopLeftRadius": 0,
                "flexDirection": "row",
                "justifyContent": "center",
                "paddingVertical": 8,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {},
                Object {
                  "opacity": 1,
                },
              ]
            }
          >
            <Text />
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "borderBottomLeftRadius": 5,
              "borderBottomRightRadius": 5,
              "borderColor": "#dfdfdf",
              "borderTopLeftRadius": 5,
              "borderTopRightRadius": 5,
              "borderWidth": 1,
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            },
            Object {
              "alignItems": "center",
              "borderTopLeftRadius": 0,
              "borderTopRightRadius": 0,
              "justifyContent": "center",
              "position": "absolute",
              "width": "100%",
            },
            Object {
              "backgroundColor": "#fafafa",
            },
            Object {
              "borderWidth": 0,
              "display": "none",
              "position": "relative",
            },
            Object {
              "maxHeight": 150,
              "top": undefined,
              "zIndex": 5000,
            },
          ]
        }
      >
        <RCTScrollView
          nestedScrollEnabled={true}
          style={
            Object {
              "width": "100%",
            }
          }
        >
          <View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Italian
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Asian
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Chipper
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Chinese
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    American
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Bakery
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Breakfast
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "opacity": 1,
                    "paddingVertical": 8,
                    "width": "100%",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "alignContent": "center",
                      "flexDirection": "row",
                      "justifyContent": "flex-start",
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {},
                        false,
                        Object {},
                      ]
                    }
                  >
                    Mexican
                  </Text>
                </View>
              </View>
            </View>
          </View>
        </RCTScrollView>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#c44536",
        "borderRadius": 10,
        "height": 50,
        "justifyContent": "center",
        "opacity": 1,
        "padding": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "alignItems": "center",
          "color": "#EBF38B",
          "fontFamily": "MonM",
          "fontSize": 38.57142857142857,
          "textAlign": "center",
          "width": 321.42857142857144,
        }
      }
    >
      Submit
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "borderColor": "black",
        "borderWidth": 1,
        "flexDirection": "row",
        "fontFamily": "MonM",
        "justifyContent": "center",
        "marginTop": 42.857142857142854,
        "opacity": 1,
        "padding": 21.428571428571427,
        "textAlign": "center",
      }
    }
  >
    <Text />
    <Text
      onPress={[Function]}
      style={
        Object {
          "paddingLeft": 21.428571428571427,
        }
      }
    >
      Log Out
    </Text>
  </View>
  <Text
    onPress={[Function]}
    style={
      Object {
        "bottom": 0,
        "color": "red",
        "fontFamily": "MonM",
        "fontSize": 30,
        "justifyContent": "flex-end",
        "marginBottom": 85.71428571428571,
        "position": "absolute",
      }
    }
  >
    Delete Account
  </Text>
  <View>
    <Modal
      animationType="none"
      hardwareAccelerated={false}
      onHide={[Function]}
      renderToHardwareTextureAndroid={true}
      transparent={true}
      visible={false}
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "black",
            "bottom": 0,
            "left": 0,
            "opacity": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      />
    </Modal>
  </View>
</View>
`;
